index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>confrince</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <section class="cover">
+      <nav class="header">
+        <img
+          src="./imges/icons/dropdown-menu.svg"
+          alt="dropdown-menu"
+          class="dropdown-menu display-non-mobile"
+        />
+        <div class="display-non-desktop disktop-nav">
+          <div class="close-btn">&#x2715;</div>
+          <a href="./index.html">
+            <img
+              src="./imges/icons/header-logo.png"
+              alt="header-icon"
+              class="logo"
+            />
+          </a>
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="./index.html">program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">News</a></li>
+          </ul>
+        </div>
+      </nav>
+      <h1>
+        "hello! sharing world!" <br />
+        <span>
+          creatve commons <br />
+          global summit <br />
+          2021
+        </span>
+      </h1>
+
```

Comment:
- [ ] Please make sure to remove any blank lines and indentations through your files of code. Kindly remember that we only add them for good reasons such as separating two logical pieces of code or two large sections apart.

_**You can find about this point and other useful notes right [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).**_

_**Please consider this point in similar situations through the files of code.**_